src/Refactoring-UI/ReGenerateEqualAndHashDriver.class.st

```diff
@@ -6,93 +6,20 @@ It uses the `RBGenerateEqualHashTransformation`.
 "
 Class {
 	#name : 'ReGenerateEqualAndHashDriver',
-	#superclass : 'RBInteractionDriver',
-	#instVars : [
-		'targetClass',
-		'selectedVariables'
-	],
+	#superclass : 'ReGenerateMethodDriver',
 	#category : 'Refactoring-UI-Drivers',
 	#package : 'Refactoring-UI',
 	#tag : 'Drivers'
 }
 
-{ #category : 'factory method' }
+{ #category : 'displaying' }
 ReGenerateEqualAndHashDriver class >> refactoringClass [
 
 	^ ReGenerateEqualHashTransformation
 ]
```

Comment:
What is the reason for storing class name on class side?

Reply:
This is to avoid `MyClass new` to get `refactoringClass`.

Reply:
Can I set this as Resolved so we can move on?

Reply:
yes